tools/project-version.sh

```diff
@@ -227,4 +231,10 @@ while IFS=" " read -r C F; do
 	sedver "$BASE/projects/$SLUG/$F" "$PAT" "$VERSION" "version constant $C"
 done < <(jq -r '.extra["version-constants"] // {} | to_entries | .[] | .key + " " + .value' "$FILE")
 
+# Update other dependencies
+
+if ! $SKIP_INTRA_MONOREPO_DEPS; then
+	"$BASE/tools/check-intra-monorepo-deps.sh" "-u"
```

Comment:
You should only do this if a version update actually happened. Not when `-c` was specified, and ideally not if all the version numbers were already up to date.

Also, a nitpick: There's no need to quote the "-u", there's no variable expansion involved in it.

```suggestion
	"$BASE/tools/check-intra-monorepo-deps.sh" -u
```